content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -116,29 +113,27 @@ sensuctl event list
+The sensu-go-sandbox keepalive event has status 0, which means the agent is in an OK state and is able to communicate with the Sensu backend.
 
-We can also see the event and the entity in the [Sensu dashboard](http://localhost:3002).
-Log in to the dashboard as the default admin user: username `admin` and password `P@ssw0rd!`.
+You can also see the event and the entity in the [Sensu dashboard](http://localhost:3002). Log in to the dashboard with the default admin credentials: username `admin` and password `P@ssw0rd!`.
 
 ## Lesson \#2: Pipe keepalive events into Slack
 
-Now that we know the sandbox is working properly, let's get to the fun stuff: creating a workflow.
-In this lesson, we'll create a workflow that sends keepalive alerts to Slack.
-(If you'd rather not create a Slack account, you can skip ahead to [lesson 3](#lesson-3-automate-event-production-with-the-sensu-agent).)
+Now that you know the sandbox is working properly, let's get to the fun stuff: creating a workflow. In this lesson, you'll create a workflow that sends keepalive alerts to Slack.
+
+_**NOTE**: If you'd rather not create a Slack account, you skip ahead to [Lesson \#3](#lesson-3-automate-event-production-with-the-sensu-agent)._
 
 **1. Get your Slack webhook URL**
 
-If you're already an admin of a Slack, visit `https://YOUR WORKSPACE NAME HERE.slack.com/services/new/incoming-webhook` and follow the steps to add the Incoming WebHooks integration, choose a channel, and save the settings.
-(If you're not yet a Slack admin, start [here](https://slack.com/get-started#create) to create a new workspace.)
-After saving, you'll see your webhook URL under Integration Settings.
+[Create a Slack workspace][2] (or use an existing workspace, if you’re already a Slack admin).
+
+Then, visit `YOUR-WORKSPACE-NAME.slack.com/services/new/incoming-webhook`. Follow the steps to add the *Incoming WebHooks* integration and save your webhook. Your webhook channel and URL will be listed under Integration Settings--you’ll need both later in this lesson.
 
 **2. Register the Sensu Slack handler asset**
 
-[Assets][2] are shareable, reusable packages that make it easy to deploy Sensu plugins.
-In this lesson, we'll use the [Sensu Slack handler asset][1] to power a `slack` handler.
+[Assets][3] are shareable, reusable packages that make it easy to deploy Sensu plugins. In this lesson, we'll use the [Sensu Slack handler asset][4] to power a `slack` handler.
 
-Use sensuctl to register the [Sensu Slack handler asset][1].
+Use sensuctl to register the [Sensu Slack handler asset][4].
 
 {{< highlight shell >}}
 sensuctl asset create sensu-slack-handler --url "https://assets.bonsai.sensu.io/3149de09525d5e042a83edbb6eb46152b02b5a65/sensu-slack-handler_1.0.3_linux_amd64.tar.gz" --sha512 "68720865127fbc7c2fe16ca4d7bbf2a187a2df703f4b4acae1c93e8a66556e9079e1270521999b5871473e6c851f51b34097c54fdb8d18eedb7064df9019adc8"
```

Comment:
When I run
```
sensuctl asset create sensu-slack-handler --url "https://assets.bonsai.sensu.io/3149de09525d5e042a83edbb6eb46152b02b5a65/sensu-slack-handler_1.0.3_linux_amd64.tar.gz" --sha512 "68720865127fbc7c2fe16ca4d7bbf2a187a2df703f4b4acae1c93e8a66556e9079e1270521999b5871473e6c851f51b34097c54fdb8d18eedb7064df9019adc8"
```

I get `Error: resource already exists` (is this somehow happening because I ran through the "Learn Sensu in 15 minutes" Katacoda scenario? Or is it another issue?)

Reply:
Attempting to reproduce this, having some issues with vagrant at the moment. I'll come back to this.

Reply:
I do not get the same error, so I would assume you already had an asset named `sensu-slack-handler` and sensuctl gave the error that it can't make it as something named that already existed:

```[vagrant@sensu-centos ~]$ sensuctl asset create sensu-slack-handler --url "https://assets.bonsai.sensu.io/3149de09525d5e042a83edbb6eb46152b02b5a65/sensu-slack-handler_1.0.3_linux_amd64.tar.gz" --sha512 "68720865127fbc7c2fe16ca4d7bbf2a187a2df703f4b4acae1c93e8a66556e9079e1270521999b5871473e6c851f51b34097c54fdb8d18eedb7064df9019adc8"
Created```

Reply:
I'm sure you're correct. It's probably from when Cameron walked me through the website quickstart...I don't know why I thought the Katacoda in-browser tutorial would have done it. Thank you!